### PR TITLE
README updated by adding required information

### PR DESCRIPTION
**Pull request by mohammadreza-ebrahimi**
_Dear Dr. Ebadi_,

I have added the my information according to  **a, b, c, d** in the **People** table . I appreciate you if you merge my branch `mohammadreza-ebrahimi` to your main.  

Please tell me if any changes is needed. 
Thank you so much

Best regards
_Mohammadreza_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,3 +17,4 @@ Topics:
 First Name | Last Name | Education | Interests
 --- | --- | --- | ---
 Javad | Ebadi | **Physics, PhD** | Physics, Programming, Data Science
+Mohammadreza | Ebrahimi | **Physics, M. Sc** | High Energy Physics, Machine Learning, Data Science
```
Comment (javadebadi) [inline README.md]: The **M. Sc** has extra space. The correct one is **M.Sc** or **MSc** or **MS**. Please replace that with one of these.

**Review by javadebadi — APPROVED** [review]
Thank you Mohammadreza for your contribution.
I will merge your changes after you apply one change  I mentioned.

**Review by javadebadi — APPROVED** [review]

**Comment by mohammadreza-ebrahimi** [comment]
I edited the issue you pointed out . 
Thank you ⚘

**Comment by javadebadi** [comment]
I merged your pull request.
Thank you very much.

**Pull request merged**